src/lib.rs

```diff
@@ -108,6 +108,15 @@ mod arch {
             .context("unable to call perf script")?
             .stdout)
     }
+
+    pub fn set_collapse_options(
```

Comment:
I'd probably prefer not to have this as a separate method -- I think we could just inline this code?

Reply:
it should be under cfg attribute conditional. How can I inline a milti-line block under `#cfg` attribute?

Reply:
`#[cfg()] { /* your code goes here */ }`

Reply:
fixed